core/src/test/resources/log4j.properties

```diff
@@ -14,5 +14,8 @@
 # limitations under the License.
 ##
 
-log4j.rootLogger=OFF
+log4j.rootLogger=INFO, stdout
```

Comment:
I don't have a strong opinion here. I wouldn't worry much in this repo anyway -- rest-utils doesn't see a lot of changes these days.